.github/ISSUE_TEMPLATE/general-purpose.md

```diff
@@ -0,0 +1,19 @@
+---
+name: General Purpose
+about: This template serves as a structured format for users to provide detailed information
+  about encountered bugs.
+title: 'triage: '
+labels: 'status: new'
+assignees: ''
+
+---
+
+<!--Thanks for taking the time to fill out this bug report! -->
+
+## Description
+> Please provide a detailed description of the bug. Attach any relevant screenshots, error messages, or files that can help understand the bug.
```

Comment:
But now this is written at it is _only_ for bugs.

Also, there isn't much structure, just "Description". Then we could just go without any template at all.

Actually I think that the structure with the following is quite reasonable for a general purpose template:

- Description: Describe your problem, use case, bug or other issue.
- Possible solution (optional): Add details on how you think this issue could be addressed.
- Alternative solutions (optional): Add details on any alternative solutions you can think of.
- Additional context (optional): Add any further, relevant information, e.g., references to other issues.